packages/panw_cortex_xdr/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.6.1"
```

Bump up the minor version
```suggestion
- version: "1.7.0"
```